src/igvfd/tests/test_audit_primary_cell.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+
+def test_audit_primary_cell_age(
+    testapp,
+    primary_cell
+):
+    res = testapp.get(primary_cell['@id'] + '@@index-data')
```

Comment:
I think should call `@@audit` here instead of `@@index-data`?

Reply:
we've always used `@@index-data`, even in encoded. Should we be switching all to `@@audit`?

Reply:
Yes, I don't see a reason to use `@@index-data` here.